plugins/node/opentelemetry-instrumentation-pg/package.json

```diff
@@ -36,7 +36,7 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=8.12.0"
+    "node": ">=14.0.0"
```

Comment:
Right - thought OTel JS dropped support for older Node versions, isn't that the case for instrumentations as well?

Reply:
We'll talk about this at SIG since I think everyone should be on the same page here. I think actually you're right since the base instrumentation class doesn't support old node versions

Reply:
Removed the change from this PR so the Node version change can be done for all instrumentations at once.